athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/rdf/NeptuneSparqlConnection.java

```diff
@@ -106,11 +105,7 @@ public Map<String, Object> next(boolean trimURI)
             Value val = bindingSet.getValue(varName);
             String sval = val.stringValue();
             Object oval = sval;
-            if (this.trimURI && val instanceof URI) {
```

Comment:
does this mean we don't support URI anymore?

Reply:
no, URI will be covered under IRI.